libs/design-system/src/ThemeProvider.tsx

```diff
@@ -68,22 +68,22 @@ export function ThemeProvider({ children }: { children: ReactNode | ReactNode[];
   const { themeStatus, setThemeStatus } = useLocalThemePreference();
 
   const toggleColorScheme = () => {
-    if (themeStatus === 'system') {
-      setThemeStatus('light');
-    } else if (themeStatus === 'light') {
-      setThemeStatus('dark');
+    if (themeStatus === ColorSchemePreferenceEnum.SYSTEM) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** This is where I personally like `switch` statements!‏